### ISSUE TITLE
Version 2 is broken on tvOS

### ISSUE THREAD

**Issue by crewtaylorfd** [DataDog/dd-sdk-reactnative#712]
### Describe the bug

tvOS does not support web views. Version 2 of the SDK requires `DatadogWebViewTracking` which does not support `tvOS`. 

### Reproduction steps

1. Create `react-native-tvos` application
2. Install version `2.0.0` of the sdk
3. `pod install`
  ```
 The platform of the target `HelloWorld-tvOS` (tvOS 13.4) is not compatible with `DatadogWebViewTracking (2.14.2)`, which does not support `tvOS`.
```

### Expected behavior

`pod install` works

### Affected SDK versions

2.0.0

### Latest working SDK version

1.9.0

### Did you confirm if the latest SDK version fixes the bug?

Yes

### Integration Methods

Yarn

### React Native Version

react-native-tvos: 0.74.3-0

**Comment by marco-saia-datadog** [comment]
Hi @crewtaylorfd! 👋 

Thank you for reaching out! We will work on a solution and I will keep you posted here.

**Comment by marco-saia-datadog** [comment]
Hi @crewtaylorfd! 👋 

We have made a simple fix on our podspec, so that the conflicting dependency is not installed when targeting tvOS.

The [PR#719](https://github.com/DataDog/dd-sdk-reactnative/pull/719) has been reviewed and merged, and it will be released soon in the next minor version of our SDK.

**Comment by marco-saia-datadog** [comment]
Closed, as the issue has been solved with [v2.4.4](https://github.com/DataDog/dd-sdk-reactnative/releases/tag/2.4.4).

**Comment by crewtaylorfd** [comment]
There is now a separate issue. `DatadogWebViewTracking` does not exist but is being imported by `DatadogSDKWrapper.swift`, `DdSdkImplementation.swift`, and `DdSdkNativeInitialization.swift` so the build fails. 
@marco-saia-datadog
